packages/core/src/sessionManagement.ts

```diff
@@ -7,7 +7,9 @@ export const EXPIRATION_DELAY = 15 * utils.ONE_MINUTE
 
 export interface Session<T> {
   renewObservable: Observable<void>
+
   getId(): string | undefined
+
   getType(): T | undefined
 }
```

Comment:
do you have a rule in your IDE that add these extra spaces?

Reply:
I fixed it later: I was using the default Webstorm reformat code. Now it is using the reformat prettier one. Do you want me to revert these ?

Reply:
yes please 🙂